src/eip7594/eip7594.c

```diff
@@ -615,95 +722,17 @@ C_KZG_RET verify_cell_kzg_proof_batch(
     if (ret != C_KZG_OK) goto out;
 
     ////////////////////////////////////////////////////////////////////////////////////////////////
-    // Compute aggregated columns
-    ////////////////////////////////////////////////////////////////////////////////////////////////
-
-    /* Start with zeroed out columns */
-    for (size_t i = 0; i < CELLS_PER_EXT_BLOB; i++) {
-        for (size_t j = 0; j < FIELD_ELEMENTS_PER_CELL; j++) {
-            size_t index = i * FIELD_ELEMENTS_PER_CELL + j;
-            aggregated_column_cells[index] = FR_ZERO;
-        }
-    }
-
-    /* Scale each cell's data points */
-    for (size_t i = 0; i < num_cells; i++) {
-        for (size_t j = 0; j < FIELD_ELEMENTS_PER_CELL; j++) {
-            fr_t cell_fr, scaled_fr;
-            size_t offset = j * BYTES_PER_FIELD_ELEMENT;
-            ret = bytes_to_bls_field(&cell_fr, (const Bytes32 *)&cells[i].bytes[offset]);
-            if (ret != C_KZG_OK) goto out;
-            blst_fr_mul(&scaled_fr, &cell_fr, &r_powers[i]);
-            size_t index = cell_indices[i] * FIELD_ELEMENTS_PER_CELL + j;
-            blst_fr_add(
-                &aggregated_column_cells[index], &aggregated_column_cells[index], &scaled_fr
-            );
-
-            /* Mark the cell as being used */
-            is_cell_used[index] = true;
-        }
-    }
-
-    ////////////////////////////////////////////////////////////////////////////////////////////////
-    // Compute sum of the interpolation polynomials
+    // Commmit to aggregated interpolation polynomial: [sum_k r^k interpolation_poly_k(s)]
```

Comment:
Typo and that last part is duplicate information & not all that useful here.
They can check the helper function's docs.
```suggestion
    // Commit to aggregated interpolation polynomial
```